.gitignore

```diff
@@ -4,4 +4,3 @@
 *.log
 *.gz
 *.pdf
```

Comment:
```suggestion
```
Don't you want to add a pdf ?

Reply:
the example branch is an empty branch with no `.gitignore`. this `.gitignore` only affects developers.

Reply:
Ah got it, nevermind my comment then. As you see I'm not too familiar with Github Actions ;)